airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java

```diff
@@ -112,6 +112,61 @@ public JsonNode toJdbcConfig(final JsonNode config) {
     return Jsons.jsonNode(configBuilder.build());
   }
 
+  private List<String> getAdditionalParameters(final JsonNode config) {
+    final List<String> additionalParameters = new ArrayList<>();
+    addJDBCUrlParameters(config, additionalParameters);
+    if (useSSL(config)) {
+      addSSLJDBCParameters(additionalParameters);
+    }
+    return additionalParameters;
+  }
+
+  private void addJDBCUrlParameters(final JsonNode config, final List<String> additionalParameters) {
```

Comment:
Nitpick. It's more readable to only capitalize the first letter in an acronym, as suggested in *Effective Java*. The argument is that 1) it's easier to recognize where the acronym ends, and 2) it works with tandem acronyms in one name.

In this case, both the `JDBC` and `URL` are acronyms. If we follow a consistent pattern and capitalize them all, it will be very hard to read: `addJDBCURLParameters`. `addJdbcUrlParameters` seems much better.

```suggestion
  private void addJdbcUrlParameters(final JsonNode config, final List<String> additionalParameters) {
```

Reply:
The name of the class does not follow this pattern because it was created a while ago.

Reply:
This method modifies the input argument. Can it just return a list of parameters and treat the inputs as immutable? That will make the code easier to understand.

Reply:
Good call. done